app/v2/views.py

```diff
@@ -0,0 +1,52 @@
+from flask import request, jsonify, make_response, abort
+import json 
+from flask_jwt_extended import (
+    jwt_required,
+    get_jwt_identity, create_access_token
+
+)
+from validate_email import validate_email
+from .models import User
+from . import v2
+
+
+@v2.route('/api/v2/auth/signup', methods = ['POST'])
+def create_user():
+    data = request.get_json()
+    username = data.get('username')
+    name = data.get('name')
+    email = data.get('email')
+    password = data.get('password')
+
+    if not all([username,name,email,password]):
```

Comment:
this is what the documentation for `all` says
**The all() method returns True when all elements in the given iterable are true. If not, it returns False.**
let `username = " "`. will `all([username])` return true or false??

Reply:
![image](https://user-images.githubusercontent.com/28872296/42236132-a9303aa8-7f02-11e8-8ca9-cc3193a5dc5d.png)

Reply:
![s](https://user-images.githubusercontent.com/28872296/42236251-0747e2e4-7f03-11e8-9f3d-a4ca2d9361fc.png)

Reply:
from the above image, do you think this validation method would work??

Reply:
on testing, it works. 
it first checks for blank input
then validates emails.
if all those pass it creates the account
i will check for a better method of validation

Reply:
I think I have seen what you meant.Thank you. correcting it